extensions/libraries/redcore/model/list.php

```diff
@@ -258,6 +258,9 @@ protected function populateState($ordering = null, $direction = null)
 		$app         = JFactory::getApplication();
 		$inputFilter = JFilterInput::getInstance();
 
+		// Load the parameters for frontend.
+		$params = $app->isSite() ? $app->getParams() : null;
```

Comment:
isSite is from 3.2, we need that function in RFactory because we still support Joomla 2.5 with redCORE